StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/OnSetterFunctionCalledMessageHandler.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  OnSetterFunctionCalledMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+struct OnSetterMessage<Values: Codable & Equatable>: Decodable {
+    let setter: String
+    let values: Values?
```

Comment:
🚫 **Blocker:** According to the docs, this is `value` (singular)
```suggestion
    let value: Value?
```